src/lib/Components/Lists/SavedShowItemRow.tsx

```diff
@@ -18,7 +18,8 @@ const track: Track<Props, {}> = _track
 @track()
 class SavedShowItemRow extends React.Component<Props> {
   handleTap() {
-    Switchboard.presentNavigationViewController(this, this.props.show.href)
+    console.log({ show: this.props.show })
+    Switchboard.presentNavigationViewController(this, this.props.show.href || `show/${this.props.show.id}`)
```

Comment:
This was why we were seeing some shows in the Favourites tab direct back to the Home tab.

Reply:
Ah... got it